### ISSUE TITLE
Wrong branch name in commit info text

### ISSUE THREAD

**Issue by jjimenezshaw** [andrewshadura/git-crecord#3]
I am working in a branch that is **not** `master`, and the info text to create the commit message was:
```
# Please enter the commit message for your changes. Lines starting  
# with '#' will be ignored, and an empty message aborts the commit. 
# On branch master
```
This `master` is hard coded (crecord_core.py). I know it is just an info, but it is really confusing.

**Comment by andrewshadura** [comment]
Fixed in 0a820c9, but not exactly the way I would like to.

**Comment by andrewshadura** [comment]
Since we’re using the Git’s built-in commit message feature, this is no longer relevant; an alternative solution never emerged.
